### ISSUE TITLE
Test file paths not expanded in subtests

### ISSUE THREAD

**Issue by rpeshkov** [Microsoft/vscode-go#1049]
Hi vscode-go developers! I've taken your function for expanding file paths in go test output into my plugin and found a bug there.

Steps to reproduce:

1. Create test with subtest and error output, i.e.

```
func TestIt(t *testing.T) {
	t.Run("SubTest", func(tin *testing.T) {
		tin.Error("Hi, i am not expanded")
	})
}
```

2. Launch test.

3. Look at path in output. It's not expanded

```
--- FAIL: TestIt (0.00s)
    --- FAIL: TestIt/SubTest (0.00s)
    	goissue_test.go:7: Hi, i am not expanded
```

Expected:
`goissue_test.go:7` -> `/Users/user/go/src/github.com/user/goissue/goissue_test.go:7`

**Comment by ramya-rao-a** [comment]
Thanks for reporting @rpeshkov!

This should be an easy regex fix in https://github.com/Microsoft/vscode-go/blob/master/src/goTest.ts#L298

PRs welcome

**Comment by rpeshkov** [comment]
Yep, sure.

**Comment by ramya-rao-a** [comment]
Closing as this is fixed with #1055
The fix will be out in the next update in the next few days

**Comment by ramya-rao-a** [comment]
The fix is now out in the latest update (0.6.63) to the Go extension
